src/components/DataView/DataViewTHead.tsx

```diff
@@ -15,13 +15,16 @@ import { DataViewProps } from "./DataViewTypes";
 
 const StyledWrapper = styled.thead`
 	text-align: left;
+	&>tr {
+		padding: 0px 12px;
+	};
 `
 
 const StyledTh = styled.th`
 	font-size: 14px;
 	text-align: left;
 	font-weight: 400;
-	padding: 5px 0px;
+	padding: 8px 32px 8px 0px;

```

Comment:
We shouldn't add another 32px as the columns already use one with the ".paddingRight" class.
```suggestion
	padding: 8px 0px;

```